cypress/support/pages/tasks/pageTasks.js

```diff
@@ -69,6 +70,8 @@ class Tasks {
 	static #homeworkDescription = '[data-testid="homework-description"]';
 	static #submissionComment = '[data-testid="submission-comment"]';
 	static #submissionText = '[data-testid="submission-text"]';
+	static #draftTasksTabButton =
```

Comment:
- why we need this giant class name?
- this is not a draft task selector, if its a similar selector then I'm pretty sure we already have it in this file